crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -415,18 +415,18 @@ impl SlidingSync {
     ) -> Result<(http::Request, RequestConfig, OwnedMutexGuard<SlidingSyncPositionMarkers>)> {
         // Collect requests for lists.
         let mut requests_lists = BTreeMap::new();
-        let all_lists_are_fully_loaded;
+        let set_timeout;
 
         {
             let lists = self.inner.lists.read().await;
-            let mut fully_loaded = true; // Start at `true` in case there is zero list.
+            let mut require_timeout = true; // Start at `true` in case there is zero list.
```

Comment:
Why using a temporary variable, btw? Now that `set_timeout` is a short variable name, we could override it in the loop directly, avoiding two lines of code and a cognitive overhead of one (1) temporary binding :grin:

Reply:
Yes, you're right. I didn't want to have the outer `set_timeout` to be mutable, but I've refactored this a little bit. Thanks for the notice.